util/encoding.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+// Helpers functions regarding the encoding of basic types ( integers ...)
```

Comment:
Is there a reason 'binary.LittleEndian.PutUint64', 'binary.LittleEndian.Uint64', etc won't work instead of these?